src/promptflow/promptflow/_sdk/_pf_client.py

```diff
@@ -40,6 +42,12 @@ def __init__(self, **kwargs):
         self._tools = ToolOperations()
         setup_user_agent_to_operation_context(USER_AGENT)
 
+    @classmethod
+    def get_instance(cls):
```

Comment:
why provide such public api?

Reply:
ok.
Mainly to use a single column here and avoid creating pf clients globally: 
src/promptflow/promptflow/_cli/_pf/_connection.py